go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/containers/image/v5 v5.25.0
 	github.com/davecgh/go-spew v1.1.1
+	github.com/go-jose/go-jose/v3 v3.0.3
```

Comment:
Is this okay to use v3 for replacement of v2? I know it is a bump, but just in case if there are breaking changes across releases.

Reply:
Done.